mcs/class/corlib/System.IO/DriveInfo.cs

```diff
@@ -169,14 +169,39 @@ public override string ToString ()
 		}
 
 		[MethodImplAttribute (MethodImplOptions.InternalCall)]
-		extern static bool GetDiskFreeSpaceInternal (string pathName, out ulong freeBytesAvail,
+		unsafe extern internal static bool GetDiskFreeSpaceInternal (char *pathName, int pathName_length, out ulong freeBytesAvail,
 							     out ulong totalNumberOfBytes, out ulong totalNumberOfFreeBytes,
 							     out MonoIOError error);
 
+		unsafe static bool GetDiskFreeSpaceInternal (string pathName, out ulong freeBytesAvail,
+						     out ulong totalNumberOfBytes, out ulong totalNumberOfFreeBytes,
+						     out MonoIOError error)
+		{
+			fixed (char *fixed_pathName = pathName)
```

Comment:
Use proper `{}` around the `fixed` content.

Reply:
I thought folllowing precedent but ok.